src/lib/components/table/filter.spec.ts

```diff
@@ -29,10 +32,11 @@ describe('Filter', () =>
         expect(filter.filterParameter).toEqual({ } as FilterParams);
     });
 
-    it('should emit search$ after search call', () =>
-    {
-        spyOn(filter.search$, 'next');
+    it('should emit search$ after search call', fakeAsync(() => {
```

Comment:
muss nicht fakeAsync sein. In dem Fall sind synchrone tasks.
Warum hat das davor nicht funktioniert?

Reply:
weil next nicht existiert auf observables